cmd/sebak/cmd/cmd_test.go

```diff
@@ -17,10 +18,56 @@ func TestParseFlagSelfValidators(t *testing.T) {
 	flagValidators = "https://localhost:12346?address=GDPQ2LBYP3RL3O675H2N5IEYM6PRJNUA5QFMKXIHGTKEB5KS5T3KHFA2"
 	flagKPSecretSeed = "SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM"
 	flagBindURL = "http://0.0.0.0:12345"
+
 	parseFlagsNode()
-	vs, err := parseFlagValidators("self")
-	require.Nil(t, err)
-	require.Equal(t, 1, len(vs))
-	validator := vs[0]
-	require.Equal(t, bindEndpoint.Host, validator.Endpoint().Host)
+	require.Equal(t, 2, len(localNode.GetValidators()))
+
+	parsedValidator, _ := node.NewValidatorFromURI(flagValidators)
+	validator := localNode.GetValidators()[parsedValidator.Address()]
+
+	require.Equal(t, validator.Address(), parsedValidator.Address())
+	require.Equal(t, validator.Endpoint().Host, parsedValidator.Endpoint().Host)
+	require.Equal(t, validator.Endpoint().Port(), parsedValidator.Endpoint().Port())
+}
+
+func TestAddingSelfValidators(t *testing.T) {
+	flagNetworkID = "sebak-test-network"
+	flagValidators = "https://localhost:12346?address=GDPQ2LBYP3RL3O675H2N5IEYM6PRJNUA5QFMKXIHGTKEB5KS5T3KHFA2"
+	flagKPSecretSeed = "SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM"
+	flagBindURL = "http://0.0.0.0:12345"
+
+	parseFlagsNode()
+
+	require.NotNil(t, localNode)
+	require.Equal(t, 2, len(localNode.GetValidators()))
+
+	{ // check validator added
+		var found bool
+		v, _ := node.NewValidatorFromURI(flagValidators)
+		for _, validator := range localNode.GetValidators() {
+			if v.Address() == validator.Address() {
+				found = true
+			}
+		}
+		require.True(t, found)
+	}
+
+	{ // check LocalNode added
+		var found bool
+		for _, validator := range localNode.GetValidators() {
+			if localNode.Address() == validator.Address() {
+				found = true
+			}
+		}
+		require.True(t, found)
+	}
+}
+
+func TestAddingSelfWithoutValidators(t *testing.T) {
+	flagNetworkID = "sebak-test-network"
+	flagValidators = ""
+	flagKPSecretSeed = "SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM"
+	flagBindURL = "http://0.0.0.0:12345"
+	_, err := parseFlagValidators(flagValidators)
```

Comment:
When we see this test case, empty `--validators` is allowed.
It has same meaning with only "self". I figure it's different with our original intent.
User should understand what validator is. Therefore I think it would be better to output error if `--validators` argument is empty.
How do you think? :)

Reply:
@Charleslee522 As you see, `flagValidators` is empty, so this test is for empty `--validators`. Am I miss something?

Reply:
@spikeekips I think `--validators "self"` is ok but `--validators`(empty) is not ok.
Empty validators seem like the user missed something.
How about you?

Reply:
@Charleslee522 :) You are right, this test checks if `--validators` is empty, it will be error.

Reply:
Oh sorry, I missunderstood `require.NotNil(t, err)`...
Thank you for your kind explanation :)